packages/angular/build/src/utils/server-rendering/esm-in-memory-loader/loader-hooks.ts

```diff
@@ -133,7 +138,12 @@ export async function load(url: string, context: { format?: string | null }, nex
   // need linking are ESM only.
   if (format === 'module' && isFileProtocol(url)) {
     const filePath = fileURLToPath(url);
-    const source = await javascriptTransformer.transformFile(filePath);
+    let source = await javascriptTransformer.transformFile(filePath);
+
+    if (filePath.includes('@angular/')) {
```

Comment:
What happens if the name does not include '@angular/'?

Reply:
It will be handled by the standard Node.js resolution, this is the same as what happens during `ng serve`.